pkgs/tools/security/quill-qr/default.nix

```diff
@@ -0,0 +1,53 @@
+{ bash
+, coreutils
+, fetchFromGitHub
+, gzip
+, jq
+, lib
+, makeWrapper
+, qrencode
+, stdenv
+, util-linux
+}:
+
+stdenv.mkDerivation rec {
+  pname = "quill-qr";
+  version = "e66726bda5177975bca11507214236d841eed32b";
```

Comment:
When fetching an unreleased commit from a repositoy, the `unstable-<commit-date>` formatting should be used. See https://nixos.org/manual/nixpkgs/stable/#sec-package-naming